### PR TITLE
Fixed N3K 3172 model code Q -> PQ, added photos

### PR DESCRIPTION
**Pull request by bjgroom**
Fixed existing template for C3172PQ and added photos of the device.

### CODE REVIEW

**Comment by lucatortiglione** [comment]
where did you get the photos ? I'd like to search photos for every cisco device

**Comment by bjgroom** [comment]
these photos came from our enterprise inventory - as we build our netbox
installation we will probably integrate more of our cisco device photos

File: device-types/Cisco/N3K-C3172PQ-10GE.yaml [hunk 1]
```diff
@@ -1,14 +1,18 @@
 ---
 manufacturer: Cisco
-model: Nexus 3172PQ-10GE
-slug: cisco-n3k-c3172q-10ge
-part_number: N3K-C3172Q-10GE
+model: Nexus C3172PQ-10GE
```
Comment (DanSheps) [inline device-types/Cisco/N3K-C3172PQ-10GE.yaml:3]:
```suggestion
model: Nexus 3172PQ-10GE
```
In keeping with all of the other Nexus switches, the "C" should not be part of the model name.

File: device-types/Cisco/N3K-C3172PQ-10GE.yaml [hunk 2]
```diff
@@ -117,11 +121,9 @@ interfaces:
...
   - name: Ethernet1/54
     type: 40gbase-x-qsfpp
-console-ports:
-  - name: Console
-    type: rj-45
-  - name: usb0
-    type: usb-mini-b
+  - name: mgmt0
+    type: 1000base-t
+    mgmt_only: true
```
Comment (DanSheps) [inline device-types/Cisco/N3K-C3172PQ-10GE.yaml:126]: Why was this re-ordered?

File: device-types/Cisco/N3K-C3172PQ-10GE.yaml [hunk 3]
```diff
@@ -1,14 +1,18 @@
...
+model: Nexus C3172PQ-10GE
+part_number: N3K-C3172PQ-10GE
+slug: cisco-n3k-c3172pq-10ge
 u_height: 1
 is_full_depth: false
+console-ports:
+  - name: Console
+    type: rj-45
+  - name: usb0
+    type: usb-mini-b
```
Comment (DanSheps) [inline device-types/Cisco/N3K-C3172PQ-10GE.yaml:12]: Why was this re-ordered?

**Review by DanSheps — CHANGES_REQUESTED** [review]

File: device-types/Cisco/N3K-C3172PQ-10GE.yaml (hunk 1 above)
Comment (bjgroom) [reply device-types/Cisco/N3K-C3172PQ-10GE.yaml]: "C" removed from model name

File: device-types/Cisco/N3K-C3172PQ-10GE.yaml (hunk 3 above)
Comment (bjgroom) [reply device-types/Cisco/N3K-C3172PQ-10GE.yaml]: Moved beneath interfaces block, before module-bays block

**Review by DanSheps — APPROVED** [review]

**Pull request merged**